pkg/controller/migmigration/stage.go

```diff
@@ -482,24 +482,36 @@ func (t *Task) stagePodReport(client k8sclient.Client) (report PodStartReport, e
 	}
 	report.started = true
 	for _, pod := range podList.Items {
+		initReady := true
+		for _, c := range pod.Status.InitContainerStatuses {
+			// If the init contianer is waiting, then nothing can happen.
+			if c.State.Waiting != nil {
```

Comment:
hrm, I wonder if this is enough? what if the init container failed and is on its way to restart? 

Do we also need to check for `c.State.Terminated` to check for all non-running states and return messages?

Reply:
Updated